pkgs/top-level/all-packages.nix

```diff
@@ -15588,8 +15588,6 @@ in
 
   libclxclient = callPackage ../development/libraries/libclxclient  { };
 
-  libco-canonical = callPackage ../development/libraries/libco-canonical { };
```

Comment:
Missing alias entry.

Reply:
What is an "alias entry" ?

Reply:
https://github.com/NixOS/nixpkgs/commit/a6bf99d97bbbae9fb2e52faa751d792f8f888bf7 does this do what you mentioned?

Reply:
Yes, thanks! In general we add these to pkgs/top-level/aliases.nix instead - could you please make that change as well?

Reply:
Ah, was reviewing an outdated PR version...